.github/workflows/pycafe-dashboards-in-CI.yml

```diff
@@ -0,0 +1,47 @@
+name: PyCafe Dashboards in CI
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+      - main
+
+env:
+  PYTHON_VERSION: "3.12"
+
+jobs:
+  create-links:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ env.PYTHON_VERSION }}
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install -r tools/tools_requirements.txt
+          pip install hatch
+      - name: "Build vizro-core package"
+        run: |
+          cd vizro-core
+          hatch build
+      - name: "Upload artifact"
+        uses: actions/upload-artifact@v4
+        with:
+          name: pip
+          path: vizro-core/dist/*.whl
+          retention-days: 14
+
+      - name: "Create PyCafe links"
+        env:
+          GITHUB_TOKEN: ${{ github.token }}
+          RUN_ID: ${{ github.run_id }}
+          GITHUB_REPOSITORY: ${{ github.repository }}
+          PR_NUMBER: ${{ github.event.number }}
```

Comment:
This feels like quite a hacky way to pass arguments to the Python script - can we just pass them as script arguments instead? I would actually just hardcode the examples into the Python script and pass these github bits through as arguments.